package/network/utils/comgt/files/ncm.json

```diff
@@ -63,5 +63,20 @@
 		},
 		"connect": "AT*ENAP=1,${profile}",
 		"disconnect": "AT*ENAP=0"
+	},
+	"mtk1": {
+		"initialize": [
+			"AT+CFUN=1"
+		],
+		"configure": [
+			"AT+CGDCONT=${profile},\\\"${pdptype}\\\",\\\"${apn}\\\",0,0",
+			"AT+CGACT=1,${profile}"
+		],
```

Comment:
Can you verify that AT+CGACT=1,1 is necessary? Often AT+CGDATA will do the necessary activation step as well.